libraries/AC_AttitudeControl/AC_PosControl.cpp

```diff
@@ -270,23 +270,23 @@ const AP_Param::GroupInfo AC_PosControl::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("_ANGLE_MAX", 7, AC_PosControl, _lean_angle_max, 0.0f),
 
-    // @Param: _TC_XY
-    // @DisplayName: Time constant for the horizontal kinimatic input shaping
-    // @Description: Time constant of the horizontal kinimatic path generation used to determine how quickly the aircraft varies the acceleration target
-    // @Units: s
-    // @Range: 0.25 2
-    // @Increment: 0.01
+    // @Param: _JERK_XY
+    // @DisplayName: Jerk limit for the horizontal kinimatic input shaping
+    // @Description: Jerk limit of the horizontal kinimatic path generation used to determine how quickly the aircraft varies the acceleration target
+    // @Units: m/s/s/s
+    // @Range: 1 20
+    // @Increment: 1
     // @User: Advanced
-    AP_GROUPINFO("_TC_XY", 8, AC_PosControl, _shaping_tc_xy_s, POSCONTROL_DEFAULT_SHAPER_TC),
```

Comment:
There's a slight danger here that beta testers may have modified _TC_XY or _TC_Z that had defaults of 0.25 and this could result in _JERK_XY and _JERK_Z being very low compared to their defaults of 5.  What would the behaviour be like if the JERK values were very low?  I guess the vehicle would be very slow to respond in autonomous modes including Auto, RTL, Loiter, etc?  A simple way to remove this danger is just to increase the indexes on these parameters from 8, 9 to 10,11.

Reply:
Done